.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -35,6 +41,9 @@ List here all the items you have verified BEFORE sending this PR. Please DO NOT
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] I have added an entry in `./changelog/fragments` using the [changelog tool](https://github.com/elastic/elastic-agent#changelog)
 - [ ] I have added an integration test or an E2E test
+- [ ] Does Endpoint team need to be informed of this change?
+- [ ] Does Profiling team need to be informed of this change?
+- [ ] Does Cloud defend team need to be informed of this change?
```

Comment:
```suggestion
- [ ] Does Cloud Defend team need to be informed of this change?
```

Reply:
I don't think this needs to be in the PR template. If we made it to a PR without checking first it's too late. We need to think about this before we write the code.

Reply:
I removed these and rather added it to the feature request template only